### ISSUE TITLE
Navigation Bar is always a few spaces apart from top

### ISSUE THREAD

**Issue by HyunseungLee-Travis** [brave/brave-browser#22685]
# Description - Desktop
![1](https://user-images.githubusercontent.com/78584173/166650034-1e9d0cda-0992-4930-8f30-c9ba6840ca27.png)

As seen on above image, the navigation bar is always a few spaces below the top with/without the bookmark list.
It is not a huge inconvenience but it is Brave Browser that is only doing this.

# Description - Mobile
The navigation bar moves while scrolling.

# Reproduce
1. Simply check out any site with navigation bar

# Issue
![1](https://user-images.githubusercontent.com/78584173/166650034-1e9d0cda-0992-4930-8f30-c9ba6840ca27.png)

# Expected look
The bar should be right below.

# Desktop/Mobile Version
1.40.31

# Android Device details
Install type (ARM, x86): all
Device type (Phone, Tablet, Phablet): all
Android version: 12

# Version/Channel Information:
Can you reproduce this issue with the current release? [x]
Can you reproduce this issue with the beta channel? []
Can you reproduce this issue with the nightly channel? []

# Other Additional Information
Does the issue resolve itself when disabling Brave Shields? []
Does the issue resolve itself when disabling Brave Rewards? []
Is the issue reproducible on the latest version of Chrome? []

**Comment by bsclifton** [comment]
@HyunseungLee-Travis sorry that we never got back to you on this one ☹️ But since you did close, I was curious if you were able to find a resolution?

My guess based on the above would be an extension possibly causing this. Some extensions will inject into the DOM and this can affect the page layout

**Comment by HyunseungLee-Travis** [comment]
I am not exactly sure, either.

Around May to October 2022, the issue persisted on both Brave on mobile and desktop.
But after that, it seems to have disappeared.
I did not change the extensions nor did I updated them.
It was happening on sites such as GitHub, YouTube, w3schools, e.t.c
Then, I tried inspecting the elements but couldn't select the blank area at the top on desktop.

Now, the problem disappeared around late 2022.
So, I believed the Brave Team fixed the issue.
I forgot about this issue for a while and closed them because it was back to the top again.
